tests/suite/test_tls.py

```diff
@@ -131,3 +144,41 @@ def test_tls_termination(self, kube_apis, ingress_controller_endpoint, test_name
         replace_secret(kube_apis.v1, tls_setup.secret_name, test_namespace, tls_setup.new_secret_path)
         wait_before_test(1)
         assert_gb_subject(ingress_controller_endpoint, tls_setup.ingress_host)
+
+        count_after = get_reload_count(tls_setup.metrics_url)
+        reloads = count_after - count_before
+        expected_reloads = 8
```

Comment:
this count will always be deterministic?

Reply:
This has changed in the latest updates, it now only expects a 1 or 0 depending on is dynamic loads is enabled.